Sources/Sentry/SentryOptions.m

```diff
@@ -708,4 +709,136 @@ - (void)setEnablePreWarmedAppStartTracing:(BOOL)enablePreWarmedAppStartTracing
+    [optionsDescription
+        addObject:[NSString stringWithFormat:@"profilesSampler: %@", self.profilesSampler]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"isProfilingEnabled: %@",
+                                            self.isProfilingEnabled ? @"YES" : @"NO"]];
+#        pragma clang diagnostic push
+#        pragma clang diagnostic ignored "-Wdeprecated"
+    [optionsDescription addObject:[NSString stringWithFormat:@"enableProfiling (deprecated): %@",
+                                            self.enableProfiling ? @"YES" : @"NO"]];
+#        pragma clang diagnostic pop
+#    endif // SENTRY_TARGET_PROFILING_SUPPORTED
+    [optionsDescription addObject:[NSString stringWithFormat:@"sendClientReports: %@",
+                                            self.sendClientReports ? @"YES" : @"NO"]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"enableAppHangTracking: %@",
+                                            self.enableAppHangTracking ? @"YES" : @"NO"]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"appHangTimeoutInterval: %f",
+                                            self.appHangTimeoutInterval]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"enableAutoBreadcrumbTracking: %@",
+                                            self.enableAutoBreadcrumbTracking ? @"YES" : @"NO"]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"tracePropagationTargets: %@",
+                                            self.tracePropagationTargets]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"enableCaptureFailedRequests: %@",
+                                            self.enableCaptureFailedRequests ? @"YES" : @"NO"]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"failedRequestStatusCodes: %@",
+                                            self.failedRequestStatusCodes]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"failedRequestTargets: %@",
+                                            self.failedRequestTargets]];
+#    if SENTRY_HAS_METRIC_KIT
+    if (@available(iOS 15.0, macOS 12.0, macCatalyst 15.0, *)) {
+        [optionsDescription addObject:[NSString stringWithFormat:@"enableMetricKit: %@",
+                                                self.enableMetricKit ? @"YES" : @"NO"]];
+    }
+#    endif // SENTRY_HAS_METRIC_KIT
+    [optionsDescription addObject:[NSString stringWithFormat:@"enableTimeToFullDisplayTracing: %@",
+                                            self.enableTimeToFullDisplayTracing ? @"YES" : @"NO"]];
+    [optionsDescription addObject:[NSString stringWithFormat:@"swiftAsyncStacktraces: %@",
+                                            self.swiftAsyncStacktraces ? @"YES" : @"NO"]];
+    [optionsDescription
+        addObject:[NSString stringWithFormat:@"cacheDirectoryPath: %@", self.cacheDirectoryPath]];
+    return [optionsDescription componentsJoinedByString:@"\n"];
```

Comment:
`m`: I don't know where I read this, but I think adding the class name and memory address in the description is good practice. So something like
```
<SentryOptions: 0x7f98f2e01d20, {
    debug = "YES";
   // ...
}>
```